distribution/deps.bzl

```diff
@@ -21,6 +21,5 @@ def vaticle_bazel_distribution():
     git_repository(
         name = "vaticle_bazel_distribution",
         remote = "https://github.com/vaticle/bazel-distribution",
-        commit = "b17950a7e34b203a18a53217b61c7a7f45c30f25" # sync-marker: do not remove this comment, this is used for sync-dependencies by @vaticle_bazel_distribution
+        commit = "aebf99748d81925d2d9e3008464df48b7469825a" # sync-marker: do not remove this comment, this is used for sync-dependencies by @vaticle_bazel_distribution
```

Comment:
Update to correct commit SHA (if changed) on merging https://github.com/vaticle/bazel-distribution/pull/361